### PR TITLE
Add default files to lambda layers

### PR DESCRIPTION
**Pull request by brene**
*Description of changes:*
- This places a "package.json" stub into the NodeJS Lambda Layer folder
- @akshbhu  @jhockett - I think we can refactor the code here to include all README.txt files in here

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-nodejs-function-runtime-provider/src/index.ts [hunk 1]
```diff
@@ -18,6 +18,25 @@ export const functionRuntimeContributorFactory: FunctionRuntimeContributorFactor
...
+                dependencies: {},
+                devDependencies: {},
+                scripts: {
+                  test: 'echo "Error: no test specified" && exit 1',
+                },
+                author: '',
+                license: 'ISC',
+              }),
+            },
+          ],
```
Comment (jhockett) [inline packages/amplify-nodejs-function-runtime-provider/src/index.ts:39]: I think this is fine for now, but we should look at storing these files elsewhere if this list grows.

Comment (brene) [reply packages/amplify-nodejs-function-runtime-provider/src/index.ts:39]: Agree

File: packages/amplify-category-function/src/provider-utils/awscloudformation/utils/storeResources.ts [hunk 2]
```diff
@@ -79,6 +79,17 @@ export function createLayerFolders(context, parameters) {
     moduleDirPath = path.join(layerDirPath, 'lib', runtime.layerExecutablePath);
     fs.mkdirSync(moduleDirPath, { recursive: true });
     fs.writeFileSync(path.join(moduleDirPath, 'README.txt'), 'Replace this file with your layer files');
+
+    if (!runtime.layerDefaultFiles) {
+      continue;
+    }
```
[the remark this answers is not in the document]
Comment (brene) [reply packages/amplify-category-function/src/provider-utils/awscloudformation/utils/storeResources.ts]: Done but without lodash.

File: packages/amplify-category-function/src/provider-utils/awscloudformation/utils/storeResources.ts [hunk 3]
```diff
@@ -79,6 +79,17 @@ export function createLayerFolders(context, parameters) {
...
+    for (let defaultFile of runtime.layerDefaultFiles) {
+      moduleDirPath = path.join(layerDirPath, 'lib', defaultFile.path);
+      if (!fs.pathExistsSync(moduleDirPath)) {
+        fs.mkdirSync(moduleDirPath);
+      }
```
[the remark this answers is not in the document]
Comment (brene) [reply packages/amplify-category-function/src/provider-utils/awscloudformation/utils/storeResources.ts]: done.

**Pull request merged**